.travis.yml

```diff
@@ -31,7 +32,11 @@ jobs:
     script:
       - make testcodecov
       - bash <(curl -s https://codecov.io/bash)
-
+  
```

Comment:
Please CMIIW, this would only check the doc creation step and fail, asking users to push updated swagger as part of a separate commit. Can goreleaser take care of generating the swagger docs before packaging the release? 
That way docs creation is automated and is part of the release.

Reply:
Your assumption is correct
We also plan to use github pages to host the API spec using the docs folder, hence need the doc folder(ie swagger.json) to stay updated.